.github/workflows/release_v2.yml

```diff
@@ -0,0 +1,459 @@
+name: Release
```

Comment:
This is identical to the existing workflow so it would only duplicate it (and one of them would fail when it came to uploading). Anyway we will not usually want to release v1 and v2 at the same time so the procedure needs to be different. Let's delete this for now and consider the release process at a later date.